src/scenes/OrgRouter/index.tsx

```diff
@@ -48,6 +48,8 @@ import useDeviceInfo from 'src/utils/useDeviceInfo';
 import useEnvironment from 'src/utils/useEnvironment';
 import useStateLocation from 'src/utils/useStateLocation';
 
+import ReportsRouter from '../Reports';
+
```

Comment:
this relative path might need to be updated to `src/<something>/Reports`

Reply:
Good catch, thanks! 🙌